ios/Sources/Repository/ContributorRepository.swift

```diff
@@ -22,6 +22,7 @@ public struct ContributorRepository: ContributorRepositoryProtocol, KMMRepositor
             flowWrapper: repository.contributorContents(),
             scopeProvider: scopeProvider
         )
+        .first()
```

Comment:
please let me know why this is necessary.

Reply:
These `.first()` not needed 😭 (It's my mistake)
I'll remove this.